testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/pages/LoginTotpPage.java

```diff
@@ -34,7 +34,7 @@
     @FindBy(css = "input[type=\"submit\"]")
     private WebElement submitButton;
 
-    @FindBy(id = "kc-cancel")
+    @FindBy(id = "kc-back")
```

Comment:
There is already button "back" defined on some superclass of LoginTotpPage. It will be good to remove this element from the page entirely, also remove the method "cancel" from the LoginTotpPage class and update (or eventually remove) the tests, which depends on the cancel button.